Formula/docker-ls.rb

```diff
@@ -0,0 +1,28 @@
+class DockerLs < Formula
+  desc "Tools for browsing and manipulating docker registries."
+  homepage "https://github.com/mayflower/docker-ls"
+  url "https://github.com/mayflower/docker-ls.git",
+    :tag => "v0.3.1",
+    :revision => "d80310976c9707e261e57ebfa9acf4e0b1781460"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+
+    (buildpath/"src/github.com/mayflower/docker-ls").install buildpath.children
+
+    system "go", "generate", "github.com/mayflower/docker-ls/lib"
+    system "go", "install", "github.com/mayflower/docker-ls/cli/..."
+
+    bin.install buildpath/"bin/docker-ls"
+    bin.install buildpath/"bin/docker-rm"
+  end
+
+  test do
+    system "#{bin}/docker-ls", "version"
+    assert_match /\Wlatest\W/m, pipe_output("#{bin}/docker-ls tags -r https://index.docker.io -u '' -p '' --progress-indicator=false library/busybox")
+    system "#{bin}/docker-rm", "version"
```

Comment:
Thanks for adding the tests! These `version` checks probably aren't necessary anymore.

Reply:
Thanks for the feedback. I agree, I've removed the `version` checks and updated the PR.